src/presets/unocss/index.js

```diff
@@ -0,0 +1,30 @@
+import Colors from "../../../src/props.colors.js";
+import Fonts from "../../../src/props.fonts.js";
+
+export const openPropsPreset = {
```

Comment:
sick, do the work in the module export with the js source files, very nice! much better than a build file's output